test/integration-new/README.md

```diff
@@ -60,4 +60,216 @@ Currently the package is named as `integration-new` because we already have `int
+var _ = Describe("Test pod networking with prefix delegation enabled", func() {
+	var (     
+	        // List of global variables used for the tests below
+		// The Pod labels for client and server in order to retrieve the
+		// client and server Pods belonging to a Deployment/Jobs
+		labelKey                = "app"
+		serverPodLabelVal       = "server-pod"
+		clientPodLabelVal       = "client-pod"
+		serverDeploymentBuilder *manifest.DeploymentBuilder
+		// Value for the Environment variable ENABLE_PREFIX_DELEGATION
+		enableIPv4PrefixDelegation string
+	)
+
+	JustBeforeEach(func() {
+
+		By("creating deployment")
+		serverDeploymentBuilder = manifest.NewDefaultDeploymentBuilder().
+			Name("traffic-server").
+			NodeSelector(f.Options.NgNameLabelKey, f.Options.NgNameLabelVal)
+
+		By("Set PD")
+		k8sUtils.AddEnvVarToDaemonSetAndWaitTillUpdated(f, utils.AwsNodeName,
+			utils.AwsNodeNamespace, utils.AwsNodeName,
+			map[string]string{"ENABLE_PREFIX_DELEGATION": enableIPv4PrefixDelegation})
+	})
+
+	JustAfterEach(func() {
+
+		k8sUtils.AddEnvVarToDaemonSetAndWaitTillUpdated(f, utils.AwsNodeName,
+			utils.AwsNodeNamespace, utils.AwsNodeName,
+			map[string]string{"ENABLE_PREFIX_DELEGATION": "false"})
+	})
+        // Context block is used to execute the behavior used 
+        // by Describe block under different scenarios 
+	Context("when testing TCP traffic between client and server pods", func() {
+		BeforeEach(func() {
+			enableIPv4PrefixDelegation = "true"
+		})
+                
+                // Below is example of individual spec specified by It
+		It("should have 99+% success rate", func() {
+			trafficTester := agent.TrafficTest{
+				Framework:                      f,
+				TrafficServerDeploymentBuilder: serverDeploymentBuilder,
+				...
+				ClientPodLabelKey:              labelKey,
+				ClientPodLabelVal:              clientPodLabelVal,
+			}
+
+			successRate, err := trafficTester.TestTraffic()
+			Expect(err).ToNot(HaveOccurred())
+			Expect(successRate).Should(BeNumerically(">=", float64(99)))
+		})
+	})
+
+	// Similarly we can also test for  UDP traffic between 
+        // client and server pods in another context here 
+})
+```
+
+More info can be found here https://github.com/onsi/ginkgo
+
+### Troubleshooting Test Failures
+
+- Adding focus to individual failing tests to selectively run them in case of troubleshooting. The most straightforward way would be to add an F in front of your Describe, Context, and It for the failing test. Another simple way would be to add --focus=<focus_value> in ginkgo command where the <focus_value> provided would be matached to the string description of the tests.
+- To analyse an identified failing test in depth before its resources are cleanedup, it would be good to comment out the AfterEach or JustAfterEach blocks after the failing test, and re-run the failing tests. This would allow for debugging of error immediately after it has occured and before cleanup. 
```

Comment:
I think you can revword this section to something like this. 

- Check if any deployments/pods are stuck which are launched as a part of the test
- Check if there are any unexpected env variable changes to the aws-node daemonset which might have got carried over from previous failed tests and thus affecting the current test. 
- Check if you have the correct Test Agent image, if you are adding new [test agent](https://github.com/aws/amazon-vpc-cni-k8s/tree/master/test/agent) changes 

To make debugging efforts easier, you can do the following 
1. Adding focus to individual failing tests to selectively run them in case of troubleshooting. The most straightforward way would be to add an F in front of your It for the failing test. Another simple way would be to add --focus=<focus_value> in ginkgo command where the <focus_value> provided would be matached to the string description of the tests.
2. Comment out the AfterEach or AfterSuite section which cleans up your test resources, rerun the failing test in focussed mode and inspect for any errors using pod logs or pod events